test/unit/dom/SVGElement.js

```diff
@@ -0,0 +1,46 @@
+import canPatchNativeAccessors from '../../../src/util/can-patch-native-accessors';
+import create from '../../lib/create';
+
+describe('dom: SVGElement', function () {
+  runTests('div');
+  runTests('slot');
+  runTests('host');
+  runTests('root');
+
+  function runTests(type) {
+    describe(`${type}: `, () => {
+      let host, root, slot, div, elem;
+
+      beforeEach(function () {
```

Comment:
could the contents of this `beforeEach` fn be moved inside the `it('should be polyfilled, ...)' fn below? Or move the `beforeEach` fn to be just before `runTests()` is defined?